app/src/org/commcare/android/tasks/templates/CommCareTask.java

```diff
@@ -7,28 +7,28 @@
 /**
  * @author ctsims
  */
-public abstract class CommCareTask<A, B, C, R> extends ManagedAsyncTask<A, B, C> {
+public abstract class CommCareTask<Params, Progress, Result, Receiver> extends ManagedAsyncTask<Params, Progress, Result> {
```

Comment:
Changes in this file aren't from lint warnings. Rather this is a readability change I've been meaning to do for a long time.